Sources/Misc/Concurrency/Purchases+async.swift

```diff
@@ -83,6 +83,16 @@ extension Purchases {
         }
     }
 
+    func restorePurchasesAsync() async throws -> CustomerInfo {
+        return try await withUnsafeThrowingContinuation { continuation in
+            self.restorePurchases { customerInfo, error in
+                continuation.resume(with: Result(customerInfo, error))
+            }
+        }
+    }
+
+    #if !ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
```

Comment:
Git is rendering this diff oddly, essentially what I did here was move the `func purchaseAsync(_ params: PurchaseParams)` function down in the file beneath `restorePurchasesAsync()` so it'd be inside the `#if !ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION` block